src/libsync/networkjobs.cpp

```diff
@@ -589,6 +590,42 @@ bool PropfindJob::finished()
 
 /*********************************************************************************************/
 
+AvatarJob::AvatarJob(AccountPtr account, QObject *parent)
+    : AbstractNetworkJob(account, QString(), parent)
+{
+    _avatarUrl = Utility::concatUrlPath(account->url(), QString("remote.php/dav/avatars/%1/128.png").arg(account->davUser()));
```

Comment:
Hmm, I think the pull request that is implementing the avatar images is not yet merged in core: owncloud/core#26872